### PR TITLE
chore: changed dependency `cpr` to use `cpy-cli`

### LINKED ISSUE #793: Evaluate to replace `cpr` by `cpy-cli`
Evaluate to replace [`cpr`](https://www.npmjs.com/package/cpr) by [`cpy-cli`](https://www.npmjs.com/package/cpy-cli), as only the latter seems to be in constant development (probably a copying mechanism might reach its end of innovation at some point in time, so that's perfectly fine, but hey), and there might be further reasons that we could identify through research.

### PR DESCRIPTION
**Pull request by nmerget**
## Proposed changes

closes #793

[cpr](https://www.npmjs.com/package/cpr) is outdated, replaced with [cpy-cli](https://www.npmjs.com/package/cpy-cli)

## Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [x] Refactoring (fix on existing components or architectural decisions)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Documentation Update (if none of the other choices apply)

### CODE REVIEW

File: packages/foundations/package.json [hunk 1]
```diff
@@ -20,16 +20,16 @@
...
-    "copy-build:package.json": "cpr package.json ../../build-outputs/foundations/package.json -o",
-    "copy-build:readme": "cpr README.md ../../build-outputs/foundations/README.md -o",
-    "copy-build:web": "cpr build ../../build-outputs/foundations/build -o",
-    "copy-normalize": "cpr ../../node_modules/@csstools/normalize.css/normalize.css scss/_normalize.scss -o",
-    "copy:scss": "cpr scss build/scss -o",
+    "copy-build:assets": "cpy assets ../../build-outputs/foundations",
+    "copy-build:package.json": "cpy package.json ../../build-outputs/foundations",
+    "copy-build:readme": "cpy README.md ../../build-outputs/foundations",
+    "copy-build:web": "cpy build ../../build-outputs/foundations",
+    "copy-normalize": "cpy ../../node_modules/@csstools/normalize.css/normalize.css scss --flat --rename=_normalize.scss",
```
Comment (mfranzke) [inline packages/foundations/package.json:31]:
```suggestion
    "copy-normalize": "cpy ../../node_modules/@csstools/normalize.css/normalize.css scss --rename=_normalize.scss",
```

It's only one file, does that really need to the `flat` option?

**Review by mfranzke — APPROVED** [review]

File: packages/foundations/package.json (hunk 1 above)
Comment (nmerget) [reply packages/foundations/package.json:31]: Sadly yes. The name of the file and directory is`normalize.css`, causing problems for `cpy`

File: packages/foundations/package.json [hunk 2]
```diff
@@ -20,16 +20,16 @@
     "build:01_normalize": "npm run copy-normalize",
     "build:02_copy": "npm run copy:scss",
     "build:03_css": "sass --no-source-map --load-path=node_modules/ --load-path=../../node_modules/ build/scss/:build/css/ --future-deprecation=import",
-    "build:04_tailwind": "cpr tailwind build/tailwind -o",
+    "build:04_tailwind": "cpy tailwind/** build/tailwind",
```
[the remark this answers is not in the document]
Comment (nmerget) [reply packages/foundations/package.json]: fixed

**Review by mfranzke — APPROVED** [review]

File: showcases/patternhub/styles/globals.scss [hunk 3]
```diff
@@ -401,8 +399,8 @@ h6 {
 
 		.backdrop {
 			border-radius: variables.$db-border-radius-sm;
-			background-color: dialog-init.$backdrop-color;
-			opacity: dialog-init.$backdrop-opacity-strong;
+			background-color: colors.$db-current-color-enabled;
+			opacity: 0.64;
```
Comment (mfranzke) [inline showcases/patternhub/styles/globals.scss:403]: @nmerget que ?

**Review by mfranzke — APPROVED** [review]

File: showcases/patternhub/styles/globals.scss (hunk 3 above)
Comment (nmerget) [reply showcases/patternhub/styles/globals.scss:403]: There was an issue with the `dialog-init` import...

**Pull request merged**